components/alert.css

```diff
@@ -0,0 +1,82 @@
+@import url(/main.css);
+@import url(/components/badges.css);
+
+.alerts {
+  padding: 1rem;
+  margin: 0.5rem;
+  background-color: white;
+  border-radius: 5px;
+}
+
+h6 {
+  padding: 1rem 0.5rem;
+}
+.alert-primary {
+  display: flex;
+  width: 600px;
```

Comment:
just a suggestion if it's a greater value then it is preferred to use rems over px